flow/scheduling/base.py

```diff
@@ -159,10 +159,12 @@ def _call_submit(submit_cmd, script, pretend):
             tmp_submit_script.flush()
             submit_cmd.append(tmp_submit_script.name)
             try:
-                subprocess.check_output(submit_cmd, universal_newlines=True)
+                subprocess.check_output(
+                    submit_cmd, stderr=subprocess.STDOUT, universal_newlines=True
+                )
             except subprocess.CalledProcessError as error:
                 raise SubmitError(
-                    f"Error when calling submission command {submit_cmd_string}: {error.output()}"
+                    f"Error when calling submission command {submit_cmd_string}:\n{error.output}"
```

Comment:
The changes in this file fix a bug that is unrelated to Bridges-2. The error output was incorrectly handled, and this had not been tested. (The bug has been in signac-flow for a long time, as far as I could tell.)